src/publish.rs

```diff
@@ -528,8 +528,7 @@ where
             info!("start hog's publish stream : {:?}", record_type);
         }
         NodeType::Crusher => {
-            // crusher's policy Id always exists.
-            let id = msg.id().unwrap();
+            let id = msg.id().expect("crusher's policy Id always exists");
```

Comment:
Could you explain why this always exists?

Reply:
- The `id` method's value is a method that gets the value of the id field of the `RequestCrusherStream` structure that cruhser sends to giganto.
- When crusher sends the value of that structure to giganto, it always includes the id value, so the value always exists.

```rust
pub struct RequestCrusherStream {
    pub start: i64,
    pub id: String,
    pub src_ip: Option<IpAddr>,
    pub dst_ip: Option<IpAddr>,
    pub sensor: Option<String>,
}
```

Apart from this issue, the return value was Option because we used to call one method, `source_id`, to get the source or id values from the `RequestStreamMessage` trait, but now we have separated them into `sensor` and `id`.  `Id` always has a value, so I think we can remove Option from the return value of the `id` method of the `RequestStreamMessage` trait in the future.